common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -5,7 +5,7 @@
 from __future__ import absolute_import
 
 import copy
-import json
+import simplejson as json
```

Comment:
This dependency should be added to `requirements/edx/base.in`, with `make upgrade` run afterwards to update the requirements files.  We happen to be installing it right now because two of our other dependencies also depend on it, but I don't want it to risk it not being installed if those packages change their implementation.